### PR TITLE
feature: Bootstrap Job: Service Account + Annotation support

### PR DESCRIPTION
**Pull request by suitupalex**

### CODE REVIEW

File: templates/bootstrap-job.yml [hunk 1]
```diff
@@ -9,6 +9,8 @@ metadata:
     heritage: {{ .Release.Service }}
   name: {{ template "metrics_enterprise.fullname" . }}-bootstrap
   namespace: {{ .Release.Namespace }}
+  annotations:
+    {{- toYaml .Values.bootstrapJob.annotations | nindent 4 }}
```
Comment (jdbaldry) [inline templates/bootstrap-job.yml:13]: I think we need a default value in `values.yaml` file for this. `bootstrap.Job.annotations: {}` and if you don't mind updating the 'Chart Values' table in the `README.md` that would be brilliant.

**Review by jdbaldry — COMMENTED** [review]
LGTM, just one small change requested.

**Comment by suitupalex** [comment]
Updated!

**Comment by jdbaldry** [comment]
Awesome, thanks! Thanks for catching

**Pull request merged**